fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -617,3 +622,21 @@ def _try_consume_work_handle(self) -> None:
             work_handle = self._work_handles.popleft()
             if work_handle.callback is not None:
                 work_handle.callback()
+
+    # Flush all the buckets, just in case
+    def _flush_buckets(self) -> None:
```

Comment:
nothing terribly complicated, go through the buckets (this will be run last), flush the ones which were not sent